### ISSUE TITLE
When use the tree-based move, some links are rewritten by another user.

### ISSUE THREAD

**Issue by tanakakz-alpha** [michitux/dokuwiki-plugin-move#114]
When I moved multiple pages with a tree based move, it was recorded that the link change of some pages was done by another user.
In the meantime I confirmed that there was access from the target user.
I have not granted edit privileges to that user.

**Comment by michitux** [comment]
This is expected behavior. By default, links are not immediately changed but only metadata which changes need to made is saved. The first user who visits the page then triggers the change. However, when you check the "Rewrite links right after the move completed"-checkbox in the tree based move interface, this should not happen.

**Comment by tanakakz-alpha** [comment]
I checked the "Rewrite links right after the move completed"-checkbox in the tree based move interface.
Is it sometimes occurred if it is executed when the user is accessing?

**Comment by michitux** [comment]
Yes, this is possible. The changes that concern a page are first collected and then only executed after everything finished. When the user accesses an affected page in between, the already collected changes are already executed and the result is saved (as an edit by that user).

**Comment by tanakakz-alpha** [comment]
I understood that it was expected behavior.

Thank you.

**Comment by qwe852147** [comment]
Sorry.I have same problem .
It possible save user name who use tree move to temp until a page are everything finished?
Then  editor will be the origin user and  not the user who  accesses an affected page in between?
